app/controllers/movement/MovementSummaryController.scala

```diff
@@ -57,26 +61,42 @@ class MovementSummaryController @Inject()(
 
   def submitMovementRequest(): Action[AnyContent] = authenticate.async { implicit request =>
     customsCacheService.fetchMovementRequest(movementCacheId, request.user.eori).flatMap {
-      case Some(data) =>
+      case Some(data) => {
+        val eoriVal = request.user.eori
+        val ducrVal = ""
+        val mucrVal = data.masterUCR
+        val movementType = "EAL"
+
         val metricIdentifier = getMetricIdentifierFrom(data)
         exportsMetrics.startTimer(metricIdentifier)
 
         customsInventoryLinkingExportsConnector
           .sendMovementRequest(movementCacheId, data.toXml)
-          .map {
-            case accepted if accepted.status == ACCEPTED =>
-              exportsMetrics.incrementCounter(metricIdentifier)
-              Redirect(controllers.movement.routes.MovementSummaryController.displayConfirmation())
+          .flatMap {
+            case accepted if accepted.status == ACCEPTED => {
```

Comment:
My propose to change the name to `response`.

It's easier to understand `response.status == ACCEPTED` than `accepted.status ...`

Reply:
This code was already there I will change as I agree it is easier to read